keylime/tpm/tpm_main.py

```diff
@@ -289,7 +289,8 @@ def __create_ek(self, asym_alg=None):
                     "-P",
                     owner_pw,
                 ]
-            elif self.tools_version == "4.2":
+            else:
+                # version 4.2 or later
```

Comment:
Conflict with #1173

Reply:
Yes, the fix on the other branch would cause the agent to crash when tpm2-tools version is > 4.2